aria_data_loaders/experiments/test_owlvit.ipynb

```diff
@@ -0,0 +1,3503 @@
+       "8   green cactus decoration    0.349122  [227, 314, 301, 401]   \n",
+       "10  green cactus decoration    0.300254  [230, 318, 313, 407]   \n",
+       "9   green cactus decoration    0.337754  [230, 320, 305, 404]   \n",
+       "\n",
+       "    graspability_score                                        device_pose  \\\n",
+       "0             0.179814  [-1.000922384490722, 1.613926143470341, 0.3042...   \n",
+       "7             0.151688  [-0.7189710509211421, 1.5494556218016742, 0.08...   \n",
+       "8             0.152204  [-0.720735162412131, 1.550295481369444, 0.0924...   \n",
+       "10            0.161941  [-0.7090865150950859, 1.5495296508035108, 0.09...   \n",
+       "9             0.159196  [-0.71687670554554, 1.551167672039137, 0.09536...   \n",
+       "\n",
+       "                                          object_pose frame_id det_uuid  \\\n",
+       "0   [-0.713391224936624, 1.4776638467529708, -0.04...       11     3520   \n",
+       "7   [-0.6356707611356129, 1.231287147329323, -0.34...      146     3153   \n",
+       "8   [-0.6330182718705579, 1.2377985441819779, -0.3...      147     3154   \n",
+       "10  [-0.586389212365872, 1.242028800435391, -0.323...      149     3156   \n",
+       "9   [-0.615217154538316, 1.2426583636878592, -0.33...      148     3155   \n",
+       "\n",
+       "     image_path  ep_id  corrected_score  dist_device_object  \n",
+       "0    218_11.jpg  218.0         0.281888            0.469275  \n",
+       "7   203_146.jpg  203.0         0.268095            0.547663  \n",
+       "8   203_147.jpg  203.0         0.268016            0.545248  \n",
+       "10  203_149.jpg  203.0         0.264820            0.533719  \n",
+       "9   203_148.jpg  203.0         0.262596            0.540108  "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>object_name</th>\n",
+       "      <th>confidence</th>\n",
+       "      <th>bbox</th>\n",
+       "      <th>graspability_score</th>\n",
+       "      <th>device_pose</th>\n",
+       "      <th>object_pose</th>\n",
+       "      <th>frame_id</th>\n",
+       "      <th>det_uuid</th>\n",
+       "      <th>image_path</th>\n",
+       "      <th>ep_id</th>\n",
+       "      <th>corrected_score</th>\n",
+       "      <th>dist_device_object</th>\n",
+       
```

Comment:
do we need this file?

Reply:
I thin it;d be good to keep this, it implements benchmarking on OWL-ViT using ADT dataset. This could be useful for further experiments on this topic. I can clean-up defunct code and write an intro section which makes more sense, if you think that'd help?